packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md

```diff
@@ -7,23 +7,51 @@ section: get-started
 import './get-started.css';
 import { Divider } from '@patternfly/react-core';
 
-## PatternFly 6 alpha 2
+## PatternFly 6 beta
+We’re excited to share the next milestone in our journey to PatternFly 6–our beta release! This beta introduces more improvements and refinement, containing all major features planned for the final release. 
 
-As we continue to work towards the full release of PatternFly 6, we're excited to share an update. This release contains the final changes to the PatternFly 6 alpha. We've fixed the bugs that have been found so far and have continued to make progress towards our final PatternFly 6 vision. 
-
-Thanks for your participation in the alpha release: the beta is coming soon!
+Your continued participation, support, and feedback will help ensure that PatternFly 6 is as well-tested and bug-free as possible before the full release. 
 
 ### Expanded PatternFly 6 support
-We added PatternFly 6 styling and token support to additional areas of our design system: 
-- Extensions
-- Charts 
-
-### Addressed issues throughout the system
-We cleaned up and fixed bugs in a number of areas:
-- We removed Sass global variables in favor of our tokens.
-- We renamed variables that previously had top/right/bottom/left in the name (for example, `PaddingTop`) to instead match inline/block start/end properties (for example, `PaddingBlockStart`). This change is a result of our introduction of [RTL support.](/developer-resources/right-to-left-handbook)
-- We removed the deprecated Sass `@import` rule in favor of new recommendations, `@use` and `@forward`.
-- We fixed various bugs that were reported following the initial alpha release.
+
+We extended PatternFly 6 styles to Topology. Now, all PatternFly elements are covered for the release, and all related design resources can be found in [our PatternFly 6 Figma kit.]()
```

Comment:
This suggestion could be way off base and/or unnecessary, but if this is just referring to the patternfly stuff in topology, I wonder if there is another word to use other than "elements" to disambiguate from [PFE/patternfly elements the product](https://patternflyelements.org/). 

```suggestion
We extended PatternFly 6 styles to Topology. Now, all PatternFly components are covered for the release, and all related design resources can be found in [our PatternFly 6 Figma kit.]()
```

Reply:
hm yeah, good point! I was trying to use a term that would be inclusive of components, but also layouts/patterns/charts/etc, but I wouldn't want to confuse things with another project. Components probably works well enough here to convey that